proxies/live/apiproxy/resources/jsc/set-logging-fields.js

```diff
@@ -95,9 +95,12 @@ function getLevel(level) {
   if (level) {
     level = level.toLowerCase()
 
-    if (level.includes('aal3')) {
+    if (level.includes(3)) {
```

Comment:
I'm not sure this would match as using number 3 instead of a string. Also wondering if we should do an exact match to prevent matching 'level3' e.g. level === '3'

Reply:
completed the change